target/linux/rtl838x/base-files/etc/uci-defaults/01_dnsmasq-loopback

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Shebang should be dropped for uci-defaults.
Consider adding an "exit 0" at the end of the file.